duckari/duck.py

```diff
@@ -25,13 +26,25 @@ def decorate(func: _ListenerT) -> _ListenerT:
 
         return decorate
 
+    def command(self, name: str = None): # TODO: Add return type
+        def decorate(func: _ListenerT) -> _ListenerT:
+            self._commands[name].append(func) # You can add multiple callbacks for a command.
+            return func
```

Comment:
There should probably be a command class written instead of just using functions, allows for much more functionality now and further down the line.

Reply:
Alright, I'll work on that.